### DISCUSSION TITLE
How to request corrections to word meanings?

### DISCUSSION THREAD

**Discussion by Rott-Apple** [meetDeveloper/freeDictionaryAPI#162] [Q&A]
Terrific, very useful API, however, I have found instances where the descriptions are incorrect. How should users point these out to request corrections? One example (Seen on 1 Oct 2022):
Toilet: A covering of linen, silk, or tapestry spread over a dressing table (This is the definition of Toilet Cloth, not Toilet)

**Comment by gchang12** [comment]
I know this is old. But the API sources data from Wiktionary. You can edit it yourself, if you'd like to.

https://en.wiktionary.org/wiki/toilet#Noun

Be aware that it's a definition that isn't exactly used that often anymore.
